csl-data.json

```diff
@@ -1,431 +1,440 @@
-                    "$ref": "name-variable" 
-                } 
-            },
-            "translator": {
-                "type": "array",
-                "items": {
-                    "$ref": "name-variable" 
-                } 
-            },
-            "accessed": {
-                "id": "date-variable",
-                "type": [
+                    ]
+                },
+                {
+                    "properties": {
+                        "circa": {
+                            "type": [
+                                "string",
+                                "number",
+                                "boolean"
+                            ]
+                        },
+                        "season": {
+                            "type": [
+                                "number"
+                            ],
+                            "minimum": 1,
+                            "maximum": 4
+                        }
+                    }
+                }
+            ]
+        },
+        "nameDefinition": {
+            "type": "array",
+            "items": {
+                "anyOf": [
                     {
+                        "type": "object",
                         "properties": {
-                            "date-parts": {
-                                "type": "array",
-                                "items": {
-                                    "type": "array",
-                                    "items": {
-                                        "type": [
-                                            "string",
-                                            "number" 
-                                        ] 
-                                    },
-                                    "maxItems": 3 
-                                },
-                                "maxItems": 2 
+                            "literal": {
+                                "type": "string"
+                            }
+                        },
+                        "required": [
+                            "literal"
+                        ]
+                    },
+                    {
+                        "type": "object",
+                        "properties": {
+                            "family": {
+                                "type": "string"
+                            },
+                            "given": {
+                                "type": "string"
+                            },
+                            "dropping-particle": {
+                                "type": "string"
+                            },
+                            "non-dropping-particle": {
+                                "type": "string"
                             },
-                            "season": {
+                            "suffix": {
+                                "type": "string"
+                            },
+                            "comma-suffix": {
```

Comment:
Not sure what this is, nor `static-ordering` or `parse-names`

Reply:
static-ordering: https://github.com/Juris-M/citeproc-js/blob/master/manual/citeproc-doc.rst#non-byzantine-names
parse-names: https://github.com/Juris-M/citeproc-js/blob/master/manual/citeproc-doc.rst#names-1

Reply:
These two attributes were cruft from the early days. I've removed them from fixtures in the test suite, they can be removed from the schema.